src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,161 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDaoImpl storageDao;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    @AfterEach
+    void afterEach() {
+        storageDao = null;
+    }
+
+    @Test
+    @DisplayName("Method register(User user) "
+            + "throws UserRegistrationException if user parameter is null")
+    void registerMethodThrowsExceptionIfUserParameterIsNull() {
+        UserRegistrationException exception = assertThrows(
+                UserRegistrationException.class,
+                () -> registrationService.register(null));
+        assertAll(() -> assertThrows(UserRegistrationException.class,
+                        () -> registrationService.register(null)),
+                () -> assertEquals("User can`t be null",
+                        exception.getMessage())
+        );
+    }
```

Comment:
Your tests should have informative names - <methodUnderTest>_<state>_<expectedBehavior>